circle.yml

```diff
@@ -0,0 +1,10 @@
+## Customize dependencies
+dependencies:
+  override:
+    - npm prune
```

Comment:
@mastilver I'm not quite seeing the advantage of using `update` over `install`? `update` would bump all deps in range on every test run, potentially failing a test/lint because of an unexpected bug in a new release of a dependency. I'd rather use a service like [Greenkeeper](https://greenkeeper.io/) to bump dependencies and check for dependency-related bugs.